app/models/rating.rb

```diff
@@ -2,6 +2,10 @@ class Rating < ApplicationRecord
   belongs_to :user
   belongs_to :cocktail
 
-  validates :stars, presence: true, numericality: { only_integer: true }
-  # validates_length_of :stars, in: 0..5
+  validates :stars, presence: true,
+    numericality: {
+      only_integer: true,
+      greater_than_or_equal_to: 0,
+      less_than_or_equal_to: 5
+      }
```

Comment:
😍

Reply:
It took me a little while longer to find this... but I found it.